spec/serializers/v1/event_error_serializer_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ::V1::EventErrorSerializer do
+  subject(:serializer) { described_class.new(event_error, root_name: 'event_error') }
+
+  let(:event_error) do
+    OpenStruct.new(
+      error: { transaction_id: ['value_already_exist'] },
+      event: create(:received_event)
```

Comment:
Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call. (https://rubystyle.guide#no-trailing-params-comma)